src/frontends/paddle/tests/places.cpp

```diff
@@ -53,7 +53,7 @@ std::vector<std::string> tensor_names = {
     "save_infer_model/scale_5.tmp_0",
 };
 
-TEST(Paddle_Places, check_tensor_names) {
+TEST(Paddle_Places, DISABLED_check_tensor_names) {
```

Comment:
This commit https://github.com/meiyang-intel/openvino/commit/6fa9c1251da4346bdccf38397fcf066a4a51c9e4  can fix Paddle_Places issue in v2.5.1. Please help to merge it.

Reply:
merged